crates/turborepo-scm/src/lib.rs

```diff
@@ -3,6 +3,10 @@
 #![feature(assert_matches)]
 #![deny(clippy::all)]
 
+//! Turborepo's library for interacting with source control management (SCM).
+//! Currently we only support git. We use SCM for finding changed files,
+//! for getting the previous version of a lockfile, and to hash files.
```

Comment:
```suggestion
//! Turborepo's library for interacting with source control management (SCM).
//! We use SCM for finding changed files, getting the previous version of a lockfile,
//! and to hash files.
```

Reply:
I read it more as a list. "blah blah blah for: first thing, second thing, and last thing"

but not blocking, grammar is fun :)